misc-scripts/xref_mapping/XrefParser/EntrezGeneParser.pm

```diff
@@ -32,14 +35,11 @@ sub run {
   my $species_id   = $ref_arg->{species_id};
   my $species_name = $ref_arg->{species};
   my $files        = $ref_arg->{files};
-  my $verbose      = $ref_arg->{verbose};
-  my $dbi          = $ref_arg->{dbi};
-  $dbi = $self->dbi unless defined $dbi;
+  my $verbose      = $ref_arg->{verbose} || 0;
+  my $dbi          = $ref_arg->{dbi} || $self->dbi;
```

Comment:
I would suggest checking definedness rather than truthiness (_i.e._ using // rather than ||) here, in case of DBI it is probably harmless but unless I am mistaken setting verbosity will no longer work correctly with || if the default value gets changed to non-zero and someone tries switching it back to 0 at run time.